EIPS/eip-1418.md

```diff
@@ -1,6 +1,7 @@
 ---
 eip: 1418
 title: Blockchain Storage Rent Payment
+description: At each block, deduct an amount of value from every account based on the quantity of storage used by that account.
 author: William Entriken (@fulldecent)
 discussions-to: https://github.com/ethereum/EIPs/issues/1418
```

Comment:
@SamWilsn we need to fix eipw erroring on this. We should at most give a warning for this particular lint.